src/core/plugin/index.js

```diff
@@ -0,0 +1,12 @@
+import { isFn } from '../util/core';
```

Comment:
This used to be located in the same file as `init` mixin, so I moved it to it's own file. Better organization I think.